lib/uvm/browser/sandbox-bus-start.js

```diff
@@ -73,7 +73,7 @@
         // For compatibility with the old Chrome App
         'window', 'XMLHttpRequest',
 
-        'console', 'btoa', 'atob' // (temporary)
+        'console', 'btoa', 'atob', 'moment' // (temporary)
```

Comment:
No, there is no need for this, since libraries are explicitly added to the Scope. @kunagpal `atob` and `btoa` are globals in browser windows, that's the reason they are here.